src/Configuration/src/SpringBootEnvBase/README.md

```diff
@@ -0,0 +1,4 @@
+# Configuration SpringBoot Env .NET Configuration Provider
```

Comment:
I don't think we're consistent with having readmes for projects, so I don't think it's _required_ to be here - but if it is here it might be useful to include a sample in this file of the types of values that it is for

Reply:
OK, this shows how to use it, but I was after the _why_ it's useful/important... eg: for use with environments built more for Spring Boot applications like SCDF where configuration is provided as a json string inside a single env variable that looks like <example>